api/api.go

```diff
@@ -201,13 +278,30 @@ func (s *BlockChainAPI) GetTransactionReceipt(
 	ctx context.Context,
 	hash common.Hash,
 ) (map[string]interface{}, error) {
-	return map[string]interface{}{}, nil
+	receipt := map[string]interface{}{}
```

Comment:
The general point on this mocking of data. I would suggest you define an interface for the indexer (as discussed on our call), which you then implement one version as mock indexer that has these values. You should then use the indexer interface in the API so you don't get carried away with API containing any additional logic.

Reply:
Do you intend to address this comment in this PR or in the future PR? If not in this PR add a TODO.

Reply:
The PR has already grown quite large, so I will tackle this in a future PR. Added a TODO: https://github.com/onflow/flow-evm-gateway/pull/11/commits/6173b96a68eb345e0ec4ed0b850b24fd228bf452